README.md

```diff
@@ -1 +1,13 @@
 # spec
+
+This repository holds the official specifications and reference implementations
+for WebAssembly.
+
+At this time, the contents of this repository are under development and known
+to be incomplet and inkorrect.
+
+Participation is welcome. Discussions about new features, significant semantic
+changes, or any specification change likely to generate substantial discussion
+should take place
+[the WebAssembly design repository](https://github.com/WebAssembly/design)
+first, so that this spec repository can remain focused.
```

Comment:
When contributing, please follow our [Code of Ethics and Professional Conduct](https://github.com/WebAssembly/design/blob/master/CodeOfConduct.md).